Add Disclaimer to CONTRIBUTING.md (#1793)
OWASP/owasp-mstg: CONTRIBUTING.md

After your PR has been submitted, we will review it as quickly as possible. This typically only takes a few days, but can vary depending on the size of the PR. Small PRs require only one reviewer, while large PRs may require multiple reviewers. We will always try to give initial feedback on your PR within 14 days. If you think we have forgotten about your PR, feel free to give us a nudge after these 7 days have passed.

Once the PR has been reviewed, the reviewers might add some comments and suggested changes, usually via GitHub's ["Suggested Changes"](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request "Applying suggested changes") feature (which can be quickly accepted and commited by simply clicking on the "Commit suggestion" button). In order to boost the workflow of reviewing and merging, **the authors of the MSTG reserve the right to commit "Suggested Changes"** falling under the following categories:
- Minor changes, i.e. only affecting formatting, fixing some clear typos, new lines, markdown linting errors, etc. can be directly commited by the MSTG authors.

> - Moderate and major changes involving the content itself or bigger structural changes must be accepted and commited by the authors of the PR themselves. In case of unresponsiveness within 2 weeks the authors of the MSTG reserve the right to commit any kind of changes.

Author: ```suggestion
- Moderate and major changes involving the content itself or bigger structural changes must be accepted and committed by the authors of the PR themselves. In case of unresponsiveness within 2 weeks the authors of the MSTG might then commit any kind of changes.
```

> Note that all those changes remain transparent to the original author who can verify them any time in the PR changelog.

Reviewer: I don't like "Note that". It doesn't really add anything in most cases. I would just leave it out:

> All those changes remain transparent to the original author who can verify them any time in the PR changelog.

Or replace it with another way to emphasize or point out the relation:

> Of course, all those changes remain transparent to the original author who can verify them any time in the PR changelog.

> Note that all those changes remain transparent to the original author who can verify them any time in the PR changelog.

Author: ```suggestion
Of course, all those changes remain transparent to the original author who can verify them any time in the PR changelog.
```

> In case of general unresponsiveness within 3-4 weeks, the authors of the MSTG reserve the right to transfer, finish or even close the PR after giving a proper justification. If your PR was closed but you'd like to finish it please contact us and we'll be happy to re-open it.

Reviewer: > reserve the right

This sounds pretty legalese. I think it is a good idea to indicate what usually happens, but perhaps it would be better if this sounded less as a disclaimer.

Author: Hi @Sjord, thanks for the review. In deed, it is a disclaimer because that's not how it happens, we just wait and wait :) That's how the idea of writing a disclaimer came out. This way we can enormously accelerate the review and merging process.

Nevertheless, I agree that it might sound too legalese. What do you think about "might then" (see the two suggestions). Does it sound more friendly but still as a disclaimer we can refer to if something happens?

Author: @sushi2k, @commjoen, what do you think?

Reviewer 2: i am fine with:
- are allowed to
- have the right to
- reserver the right to
- might have to

> In case of general unresponsiveness within 3-4 weeks, the authors of the MSTG reserve the right to transfer, finish or even close the PR after giving a proper justification. If your PR was closed but you'd like to finish it please contact us and we'll be happy to re-open it.

Author: ```suggestion
In case of general unresponsiveness within 3-4 weeks, the authors of the MSTG might then transfer, finish or even close the PR after giving a proper justification. If your PR was closed but you'd like to finish it please contact us and we'll be happy to re-open it.
```
